src/containers/User/Detail/Credits/components/CreditHistorySection.jsx

```diff
@@ -31,16 +32,17 @@ export default ({ proposalCreditPrice, user }) => {
     PAYWALL_STATUS_PAID === userPaywallStatus
       ? [...proposalCreditsPurchases, paywallPayment]
       : proposalCreditsPurchases;
+  const policy = usePolicy();
```

Comment:
```suggestion
  const { policy } = usePolicy();
```
A good way to avoid the redundant `policy.policy.minconfirmationsrequired` prop access 😁. Unless your destruction gets too big, please use object destruction.